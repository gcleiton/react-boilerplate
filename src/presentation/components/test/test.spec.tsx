import React from 'react'
import { render, screen } from '@testing-library/react'
import Test from './testing'

describe('test', () => {
  it('it works', () => {
    render(<Test />)

    expect(screen.getByRole('heading', { name: /test/i })).toBeInTheDocument()
  })
})
