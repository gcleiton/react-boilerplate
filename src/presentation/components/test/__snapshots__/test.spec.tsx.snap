// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test it works 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  color: purple;
  font-size: 2rem;
}

<h1
  class="c0"
>
  test
</h1>
`
